cfn/configs/klaxon-scheduled-task/prod/us-east-1/config.yml

```diff
@@ -27,15 +27,36 @@ _globals:
   - !_join &CLUSTER_NAME ["-", *CLUSTER_PREFIX, *ENVIRONMENT]
   - !_join &STACK_NAME ["-", *CLUSTER_NAME, *SERVICE_NAME]
 
-template: cfn/ecs-task/templates/ecs-task.template.yml
-stack_name: *STACK_NAME
-region: *REGION
+template: cfn/templates/ecs-task.template.yml
+stack_name: *STACK_NAME # newsroom-prod-klaxon-scheduled-task
+region: *REGION # us-east-1
 
 context:
-  debug: *DEBUG
-  name: *SERVICE_NAME
-  environment: *ENVIRONMENT
-  cluster_name: *CLUSTER_NAME
-  family: email-prod-ecs
-  image_name: ${AWS::AccountId}.dkr.ecr.${AWS::Region}.amazonaws.com/klaxon:prod-latest
-  schedule_expression: "*/10 * * * *"
\ No newline at end of file
+  debug: *DEBUG # false
+  name: *SERVICE_NAME # klaxon-scheduled-task
+  cluster_name: *CLUSTER_NAME # newsroom-prod
+  family: klaxon-scheduled-task-prod
+  container_name: klaxon-prod
+  image: ${AWS::AccountId}.dkr.ecr.${AWS::Region}.amazonaws.com/klaxon-rake:prod-latest
+  schedule_expression: "0/10 * * * ? *"
+  environment:
+    DATABASE_URL: "postgresql://{{resolve:secretsmanager:/klaxon/aurora-postgresql-prod/app:SecretString:username}}:{{resolve:secretsmanager:/klaxon/aurora-postgresql-prod/app:SecretString:password}}@{{resolve:secretsmanager:/klaxon/aurora-postgresql-prod/app:SecretString:writer-host}}:5432/klaxon"
```

Comment:
a suggestion to take or leave is that you could add a single `DATABASE_URL` value to secrets manager to make this bit more succinct/easier to read